src/main/webapp/jsp/instructorSearch.jsp

```diff
@@ -278,14 +278,17 @@
                                                         responseCommentIndex++;
                                                         String frCommentGiver = data.feedbackResponseCommentSearchResultBundle
                                                                                          .commentGiverTable.get(frc.getId().toString());
+                                                        if (!frCommentGiver.equals("Anonymous")) {
+                                                            frCommentGiver = frc.giverEmail;
+                                                        }
                                                 %>
                                                 <li class="list-group-item list-group-item-warning"
                                                     id="responseCommentRow-<%=fsIndx%>-<%=qnIndx%>-<%=responseIndex%>-<%=responseCommentIndex%>">
                                                     <div id="commentBar-<%=fsIndx%>-<%=qnIndx%>-<%=responseIndex%>-<%=responseCommentIndex%>">
                                                         <span class="text-muted">
                                                             From:
-                                                            <b><%=frCommentGiver%></b>
-                                                            on <%=TimeHelper.formatTime(frc.createdAt)%>
+                                                            <%= frCommentGiver %>
+                                                            [<%= frc.createdAt %>] <%=frc.getEditedAtText(frc.giverEmail.equals("Anonymous"))%>
```

Comment:
Instead of `frc.giverEmail`, I believe frCommentGiver should be used instead. The current condition will always give a false as `frc.giverEmail` will never equal "Anonymous"